packages/messenger/src/RestrictedMessenger.ts

```diff
@@ -342,7 +342,10 @@ export class RestrictedMessenger<
     EventType extends
       | AllowedEvent
       | (Event['type'] & NamespacedName<Namespace>),
-  >(event: EventType, handler: ExtractEventHandler<Event, EventType>) {
+  >(
+    event: EventType,
+    handler: ExtractEventHandler<Event, EventType> | SelectorEventHandler,
```

Comment:
I see that changes were made to RestrictedMessenger in this package but not in `base-controller`, should we copy them there too?